pkg/util/versions/versions.go

```diff
@@ -0,0 +1,31 @@
+package versions
+
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+)
+
+// CalculateFeatureCompatibilityVersion returns a version in the format of "x.y"
+func CalculateFeatureCompatibilityVersion(versionStr string) string {
+	v1, err := semver.Make(versionStr)
+	if err != nil {
+		return ""
+	}
+
+	baseVersion, _ := semver.Make("3.4.0")
+	if v1.GTE(baseVersion) {
+		ans, _ := MajorMinorVersion(versionStr)
+		return ans
```

Comment:
I think we can make it much easier. All `MajorMinorVersion` is doing is re-making the `semver` version we already have here as `v1`.

We can simply replace all of this with: `return fmt.Sprintf("%d.%d", v1.Major, v1.Minor)
`